libr/core/rtr.c

```diff
@@ -783,7 +783,7 @@ static int r_core_rtr_http_thread (RThread *th) {
 	HttpThread *ht = th->user;
 	if (!ht || !ht->core) return false;
 	ret = r_core_rtr_http_run (ht->core, ht->launch, ht->path);
-	free ((void *)ht->path);
+	free ((char *)ht->path);
```

Comment:
the path field in the HttpThread struct must be char * instead of const char *. and then you can remove the casts